server/loadConfigs.js

```diff
@@ -18,6 +18,21 @@ function loadMainConfig(options = {}) {
   return config;
 }
 
+// load main bpanel config
```

Comment:
could there be a function that is `loadConfig(name, options = {})`?

Reply:
How do you mean? What would that be used for?

Reply:
`loadMainConfig` --> `loadConfig`

Reply:
makes it more general

Reply:
I think that is a good idea

Reply:
Done. Much cleaner. Thanks 👍